### ISSUE TITLE
collective.filemeta dependency is missing for 1.2.0

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/Products.OpenXml#5]
When I startup Plone I get:

```
  File ".../Products.OpenXml-1.2.0-py2.7.egg/Products/OpenXml/utilities.py", line 19, in <module>
    from collective.filemeta.interfaces import IFileMetaProvider, IContentMetaUpdater
zope.configuration.xmlconfig.ZopeXMLConfigurationError: File "/.../instance/etc/site.zcml", line 16.2-16.23
    ZopeXMLConfigurationError: 
File "/Users/maurits/shared-eggs/Products.OpenXml-1.2.0-py2.7.egg/Products/OpenXml/configure.zcml", line 23.3-27.6
    ImportError: No module named filemeta.interfaces
```

**Comment by petri** [comment]
It should still be possible to use Products.OpenXml without collective.filemeta. Perhaps it's best to move the ['offending' utility registrations](https://github.com/collective/Products.OpenXml/blob/master/Products/OpenXml/configure.zcml#L21-L62) inside a conditional ZCML block.

**Comment by petri** [comment]
Tagged 1.2.1 with the fix.

**Comment by mauritsvanrees** [comment]
Thank you!

**Comment by petri** [comment]
Now released 1.2.1 on PyPI, too.
